src/main/webapp/app/exercises/quiz/manage/quiz-exercise-detail.component.html

```diff
@@ -133,30 +133,34 @@ <h4 class="title">
                                     [(ngModel)]="scheduleQuizStart"
                                     (ngModelChange)="cacheValidation()"
                                 />
-                                <label class="form-check-label custom-control-label" for="cbScheduleQuizStart" jhiTranslate="artemisApp.quizExercise.startTime"> </label>
+                                <label class="form-check-label custom-control-label" for="cbScheduleQuizStart" jhiTranslate="artemisApp.quizExercise.setStartTime"> </label>
+                                <jhi-help-icon placement="auto" text="artemisApp.quizExercise.startTimeExplanation"></jhi-help-icon>
                             </div>
                             <div class="form-check custom-control custom-checkbox" *ngIf="quizExercise.quizMode === QuizMode.BATCHED && false">
                                 <!-- no scheduled batched mode yet -->
                                 <jhi-button (click)="addQuizBatch()" [btnType]="ButtonType.DEFAULT" title="artemisApp.quizExercise.addBatch"></jhi-button>
                             </div>
-                            <ng-container *ngIf="quizExercise.quizBatches && quizExercise.quizMode !== QuizMode.INDIVIDUAL">
+                            <ng-container *ngIf="quizExercise.quizBatches && quizExercise.quizMode === QuizMode.SYNCHRONIZED">
```

Comment:
this whole thing could probably be simplified by removing the fragments of scheduled batched mode and always just having at most one batch for synchronized mode and none otherwise. how things are designed currently isn't really going to work for scheduling batches anyways and even now there are problems with saving batches which is why i disable that in #5079.

Reply:
As you suggested above, postponing it to a later PR sounds reasonable and then maybe we can discuss and find a solution that would be better in the long term.